Nightly build is failing

The nightly build is failing during cross platform compile. Here is the latest output from the goreleaser build.

```
release failed after 45.55s error=failed to build for linux_arm_7: # github.com/influxdata/platform/tsdb/tsm1
tsdb/tsm1/tombstone.go:469:14: constant 4278190080 overflows int
```
Build log: https://circleci.com/gh/influxdata/platform/11413

--- a/.goreleaser.yml
+++ b/.goreleaser.yml
@@ -5,17 +5,7 @@ builds:
     - darwin
     goarch:
     - amd64
-    - 386
-    - arm
     - arm64
-    goarm:
-    - 6
-    - 7
-    ignore:
-      - goos: darwin
-        goarch: 386
-      - goos: windows
-        goarch: 386
     main: ./cmd/influx/
     env:
       - GO111MODULE=on
@@ -26,17 +16,7 @@ builds:
     - darwin
     goarch:
     - amd64
-    - 386
-    - arm
     - arm64
-    goarm:
-    - 6
-    - 7
-    ignore:
-      - goos: darwin
-        goarch: 386
-      - goos: windows
-        goarch: 386
     main: ./cmd/influxd/
     flags:
       - -tags=assets

